app/tests/components_tests/test_tasks.py

```diff
@@ -295,3 +307,111 @@ def test_update_sagemaker_shim(
         _repo_login_and_run(command=["crane", "manifest", old_repo_tag])
 
     assert "MANIFEST_UNKNOWN" in error.value.stderr.decode()
+
+
+@pytest.mark.django_db
+def test_add_image_to_object(settings):
+    settings.task_eager_propagates = (True,)
+    settings.task_always_eager = (True,)
+
+    ds = DisplaySetFactory()
```

Comment:
These two tests only use display sets and not archive items? Is that tested elsewhere.

Reply:
Only indirectly (i.e. we test the archive item updating views), but I'll adjust this test for completeness.